src/android/app/src/main/java/io/github/mandarine3ds/mandarine/NativeLibrary.kt

```diff
@@ -655,6 +655,12 @@ object NativeLibrary {
             FileUtil.deleteDocument(path)
         }
 
+    @Keep
+    @JvmStatic
+    fun addNetPlayMessage(type: Int, message: String) {
```

Comment:
Is this used for something?

Reply:
It will possibly be used for the chat PR up next

Reply:
Its used in the chat